library/core/src/iter/adapters/peekable.rs

```diff
@@ -251,7 +250,7 @@ impl<I: Iterator> Peekable<I> {
     /// assert_eq!(iter.collect::<Vec<_>>(), vec![&5, &3]);
     /// ```
     #[inline]
-    #[unstable(feature = "peekable_peek_mut", issue = "78302")]
+    #[stable(feature = "peekable_peek_mut", since = "1.52.0")]
```

Comment:
```suggestion
    #[stable(feature = "peekable_peek_mut", since = "1.53.0")]
```

(To be resolved after FCP.)